internal/backends/qt/qt_window.rs

```diff
@@ -1347,7 +1347,18 @@ impl WindowAdapterSealed for QtWindow {
         let component_rc = WindowInner::from_pub(&self.window).component();
         let component = ComponentRc::borrow_pin(&component_rc);
         let root_item = component.as_ref().get_item_ref(0);
-        if let Some(window_item) = ItemRef::downcast_pin(root_item) {
+        if let Some(window_item) = ItemRef::downcast_pin::<WindowItem>(root_item) {
+            if window_item.width() <= 0. {
+                window_item.width.set(
+                    component.as_ref().layout_info(Orientation::Horizontal).preferred_bounded(),
+                )
+            }
+            if window_item.height() <= 0. {
+                window_item.height.set(
+                    component.as_ref().layout_info(Orientation::Horizontal).preferred_bounded(),
```

Comment:
```suggestion
                    component.as_ref().layout_info(Orientation::Vertical).preferred_bounded(),
```

Reply:
oops, well spotted.

(note that i think we set this for nothing because it is going to be overwritten later, but i just did it here to be consistent)